packages/neon/neon/lib/src/utils/push_utils.dart

```diff
@@ -76,15 +76,31 @@ class PushUtils {
         final cache = Cache(platform);
         await cache.init();
 
-        NextcloudNotificationsNotification? notification;
         var accounts = <Account>[];
         Account? account;
+        NextcloudNotificationsNotification? notification;
+        AndroidBitmap<Object>? largeIconBitmap;
         try {
           accounts = loadAccounts(AppStorage('accounts', sharedPreferences));
           account = accounts.find(instance);
           if (account != null) {
             notification =
                 (await account.client.notifications.getNotification(id: pushNotification.subject.nid!)).ocs.data;
+            if (notification.icon != null) {
+              final cacheManager = DefaultCacheManager();
+              final file = await cacheManager.getSingleFile(notification.icon!);
+              const iconSize = 256;
+              late Bitmap bitmap;
+              if (notification.icon!.endsWith('.svg')) {
+                final pictureInfo = await vg.loadPicture(SvgStringLoader(file.readAsStringSync()), null);
```

Comment:
Just read a bit into how the entire svg loading/rasterizing works and it looks like we need to dispose it afterwards:

```dart
/// Load the [PictureInfo] from a given [loader].
/// 
/// It is the caller's responsibility to handle disposing the picture when they are done with it.
```